src/javascript/app_2/App/Components/Elements/Media/media-icon.jsx

```diff
@@ -1,7 +1,15 @@
-import React from 'react';
+import PropTypes from 'prop-types';
+import React     from 'react';
 
-const MediaIcon = () => (
-    <div className='media__icon' />
-);
+const MediaIcon = ({ is_enabled, enabled, disabled }) => {
+    const Icon = is_enabled ? enabled : disabled;
+    return <Icon className='media__icon' />;
+};
+
+MediaIcon.propTypes = {
+    disabled  : PropTypes.func,
+    enabled   : PropTypes.func,
```

Comment:
Are `enabled` and `disabled` functions, looks like they are svg elements?

Reply:
imported svg elements only work as functions, tried `node`, `element` and `string` nothing else works on them.

Reply:
strange, `PropTypes.instanceOf(Element)` does not work? If it doesn't work perhaps we can add a comment that mentions that this component expects svg elements as parameters?